src/main/java/reactor/core/publisher/FluxReplay.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011-2016 Pivotal Software Inc, All Rights Reserved.
```

Comment:
@smaldini on `FluxReplay`: in `ReplaySubscriber#scan`, shouldn't the `CAPACITY` take unbounded buffer into account like it did before?